ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -363,6 +380,7 @@ public void setFontSize(float fontSize) {
               : (float) Math.ceil(PixelUtil.toPixelFromDIP(fontSize));
     }
     mFontSize = (int) fontSize;
+    mLetterSpacing = calculateLetterSpacing(mPreviousLetterSpacing, mFontSize);
```

Comment:
Still on the subject of unit conversions:

`mFontSize` is in pixels here, but letter spacing is given in _points_ (presumably the same units as `mFontSizeInput` - SP or DIP depending on `mAllowFontScaling`). We need to pass _ems_ to `setLetterSpacing()`; the correct calculation to yield that would seem to be `letterSpacing / mFontSizeInput` (handling `UNSET` gracefully somehow - is there a default font size defined somewhere?), since the font size is by definition the em size.